MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.google.moviestvsentiments;
+
+import android.content.Context;
```

Comment:
Please adjust import order and spacing. In Play Movies and most teams in Google, static imports are ordered before imports. (same for other files in commit) https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing